kani-driver/src/cbmc_output_parser.rs

```diff
@@ -816,9 +815,9 @@ fn has_check_failure(properties: &Vec<Property>, description: &str) -> bool {
 fn modify_undefined_function_checks(mut properties: Vec<Property>) -> (Vec<Property>, bool) {
     let mut has_unknown_location_checks = false;
     for mut prop in &mut properties {
-        if prop.description.contains(ASSERTION_FALSE)
-            && extract_property_class(prop).unwrap() == DEFAULT_ASSERTION
+        if prop.description == DEFAULT_ASSERTION
             && prop.source_location.file.is_none()
+            && prop.source_location.function.is_some()
         {
             prop.description = "Function with missing definition is unreachable".to_string();
```

Comment:
It looks like the propery_id as well as the location include the name of the missing function. Can we use that to print a nicer error message? The only issue is that the name is mangled. :(

Reply:
It's rust code now. Demangle? e.g. https://crates.io/crates/rustc-demangle

Reply:
Thanks for the suggestion! AFAIK we can use `rustc-demangle` to demangle the name. For some reason, demangling works only partially for missing function definitions, but still provides a nice result:

![Screen Shot 2022-08-12 at 13 09 09](https://user-images.githubusercontent.com/73246657/184409074-162db1e7-486c-4b7e-b6ff-963ca5fe5eec.png)

I could also rebuild the property name with the demangled function name in #1514. Can you review this PR again?